torch_sim/models/metatomic.py

```
@@ -34,27 +34,27 @@
+        This class is a placeholder for the MetatomicModel class.
+        It raises an ImportError if metatomic is not installed.
         """
 
         def __init__(self, err: ImportError = exc, *_args: Any, **_kwargs: Any) -> None:
             """Dummy init for type checking."""
             raise err
 
 
-class MetatensorModel(torch.nn.Module, ModelInterface):
-    """Computes energies for a list of systems using a metatensor model.
+class MetatomicModel(torch.nn.Module, ModelInterface):
```

Pylint (3.3.7)

[error] class already defined line 39

(E0102)